.prettierrc

```diff
@@ -1,6 +1,5 @@
 {
 	"useTabs": true,
-	"singleQuote": true,
```

Comment:
Please don't change this file because it is a standard for this project and affects code formatting for everyone. After you revert it back, please run `yarn format` to ensure the code is formatted according to the previous setup.